test/datafmt.jl

```diff
@@ -204,7 +204,7 @@ let i18n_data = ["Origin (English)", "Name (English)", "Origin (Native)", "Name
         "Russia", "Mikhail Gorbachev", "Россия", "Михаил Горбачёв",
         "Russia", "Boris Grebenshchikov", "Россия", "Борис Гребенщиков",
         "Slovenia", "\"Frane \"\"Jezek\"\" Milcinski", "Slovenija", "Frane Milčinski - Ježek",
-        "Syracuse (Sicily)", "Archimedes", "Συρακούσα", "Ἀρχιμήδης",
+        "Syracuse (Sicily)", "Archimedes", "Συρακούσα", "Ἀρχιμήδης",
```

Comment:
I made the change from the Github file editor, which I assume takes its encoding from the host browser? It seems to be UTF8.